src/main/java/uk/nhs/hee/tis/trainee/forms/model/content/LtftContent.java

```diff
@@ -50,6 +51,7 @@ public record LtftContent(
     Discussions discussions,
     CctChange change,
     Reasons reasons,
+    @With
     Person assignedAdmin) implements FormContent {
```

Comment:
I'm missing something about why assignedAdmin is handled differently from the other read-only fields (annotated here, but mainly in the service). Could you add a note to explain this please 😀

Reply:
Oh ok, I get it, its the one field in the LtftContent object that the trainee is not the 'master' of

Reply:
I'll add an in-line comment in the service 👍🏻

Reply:
I do keep circling round to whether the assigned admin should be lifted up to the parent form instead of in the content 🤔

Reply:
Yeah, it might be neater...

Reply:
I've got a separate piece of work to add the endpoint for assigning the admin, so I'll do the refactoring there rather than adding more changes to this bug fix.